tests/test_redis_aggregator.py

```diff
@@ -74,5 +75,11 @@ def on_batch_response(self, market_stats):
 selected = pv.select_aggregator(aggregator.aggregator_uuid)
 logging.info(f"SELECTED: {selected}")
 
+
+redis_market = RedisMarketClient('house-2')
+market_slot_string_2 = today().add(minutes=60).format(DATE_TIME_FORMAT)
+
 while not aggregator.is_finished:
-    sleep(0.5)
+    list_market_stats_results = redis_market.list_market_stats([market_slot_string_2])
```

Comment:
As discussed offline, this is just as reference for the aggregator user. Please move it to the on_market_cycle event and leave the while loop clean (and also revert the update time to 0.5 seconds). Thanks!